src/lib/FreeC/Pass/EtaConversionPass.hs

```diff
@@ -51,20 +98,30 @@
 module FreeC.Pass.EtaConversionPass
   ( etaConversionPass
     -- * Testing interface
+  , etaConvertFuncDecl
   , etaConvertExpr
   )
 where
 
-import           Control.Monad                  ( replicateM )
-import           Data.Maybe                     ( fromMaybe )
+import           Control.Monad                  ( replicateM
+-- TODO: Remove liftM2 import after moving minimumM to a util module 
+                                                , liftM2
+                                                )
+import           Data.Maybe                     ( fromMaybe
+                                                , fromJust
+                                                )
 
 import           FreeC.Environment
 import           FreeC.Environment.Fresh
+import           FreeC.Environment.Entry
 import           FreeC.IR.SrcSpan
 import           FreeC.IR.Subterm
 import qualified FreeC.IR.Syntax               as IR
 import           FreeC.Monad.Converter
 import           FreeC.Pass
+-- temporary import; ideally, this pass should be moved before the TypeSignaturePass.
+import           FreeC.Pass.TypeSignaturePass   ( splitFuncType )
```

Comment:
Could you make this a `TODO` comment such that we don't forget that we have to address this in the future?